crates/hulk_webots/src/camera.rs

```diff
@@ -67,8 +67,9 @@ impl Camera {
             let mut bgra_buffer = self.buffer.lock();
             self.buffer_updated.wait(&mut bgra_buffer);
             bgra_buffer
+                .clone()
```

Comment:
This behavior is different compared to the robotics implementation. I'd be fine merging this anyway and opening an issue that we should make the implementations similar. However the fix in this PR would allow people to use webots again. What do you think of this @schmidma @knoellle

Reply:
I was tempted to use the tokio sender/ receiver pattern, but I wanted to get it working in the first place to work on another PR.

Reply:
Please open an issue with a proper description of the problem, then I'm totally fine with this quick fix to get webots running again

Reply:
#1073